CMakeLists.txt

```diff
@@ -219,7 +219,8 @@ if(HIP_PLATFORM STREQUAL "hcc")
     add_library(hip_hcc SHARED ${SOURCE_FILES_RUNTIME})
     add_library(hip_hcc_static STATIC ${SOURCE_FILES_RUNTIME})
     if(HIP_COMPILER STREQUAL "hcc")
-      target_link_libraries(hip_hcc PRIVATE hc_am)
+      find_package(hcc)
+      target_link_libraries(hip_hcc PUBLIC hcc::hccrt hcc::hc_am)
```

Comment:
hip_hcc should not link against hccrt as `PUBLIC`. hccrt requires an hcc compiler and enables device compilation Adding it to hip_hcc would not allow hip_hcc to be used from an host-only compiler(such as gcc) nor using hip_hcc without device compilation.

Reply:
All of this is only done when `HIP_COMPILER` is set to `hcc`, however. I can tell you that the hip-targets-release.cmake that's generated out of this is identical to the one that we actually package (the one I removed in this commit).

Or is setting this going to change the actual compilation somewhere and result in a different set of HIP binaries?